server/src/weaverbird/backends/mongo_translator/steps/aggregate.py

```diff
@@ -0,0 +1,51 @@
+from typing import Any, Dict, List, Union
+
+from weaverbird.pipeline.steps import AggregateStep
+
+
+def translate_aggregate(step: AggregateStep) -> List[Dict[str, Any]]:
```

Comment:
The other steps are hinted with the return type `list` only.
In the front-end version, we had an alias called `MongoStep`. Maybe it could be semantically nice to have an TypeAlias here too (to make sure all steps have a similar signature).

Reply:
:+1: